src/Slider.jsx

```diff
@@ -29,7 +29,7 @@ function killEvent(ev) {
 
 class Button extends React.Component {
   render() {
-    return <button {...this.props} type="button" />;
+    return <button {...this.props} aria-label="Slider Handle" type="button" />;
```

Comment:
Ok, alternatively should we revert these to not be buttons? 

It seems like we can't have default handles, you'll always have to pass in a component in order to be a11y compliant, have i18n, etc...

Having two props is confusing and not really very default-y.